arboretum/kubernetes/README.md

```diff
@@ -17,7 +17,76 @@ to include the fetchers and checks from this library in your downstream project.
 
 ## Fetchers
 
-Fetchers coming soon...
+### Cluster Resource
+
+* Class: [ClusterResourceFetcher][fetch-cluster-resource]
+* Purpose: Write the resources of **stand-alone** Kubernetes clusters to the
+evidence locker.  **NOTE:** Do not use this fetcher for managed clusters.
+Instead use the [IBM Cloud cluster list fetcher][ibm-cloud-cluster-list-fetcher].
+* Behavior: Retrieve stand-alone Kubernetes cluster resource data for the provided
+list of clusters.  TTL is set to 1 day.
+* Configuration elements:
+  * `org.kubernetes.cluster_resources.clusters`
+    * Required
+    * List of dictionaries:
+      * `label`: An arbitrary label to specify token in credentials file
+      * `server` A URL as a string to a Kubernetes stand-alone cluster
+  * `org.kubernetes.cluster_resources.types`
+    * Optional
+    * List of resource types as strings
+      * NOTE: For core group API resources, the resource name must be in
+      _plural form_.  See example below.
+      * NOTE: For other "custom" API resources, the resource name must be in the
+      following format: `"APIGROUP/VERSION/NAME"`.  See example below.
+    * Use if looking to override the default of `["nodes", "pods", "configmaps"]`.
+    Otherwise do not include.
+* Expected configuration:
+
+  ```json
+  {
+    "org": {
+      "kubernetes": {
+        "cluster_resources": {
+          "clusters": [
+            {
+              "label": "devcluster1",
+              "server": "https://myserver1:30000"
+            }
+          ],
+          "types": ["nodes", "pods", "apigroup.example.com/v1/mycustom"]
+        }
+      }
+    }
+  }
+   ```
+
+
+* Required credentials:
+  * Kubernetes API server token with read/view permissions are needed for this fetcher to successfully retrieve the evidence.
+    The token can be extracted from local kubeconfig file (usually stored in `~/.kube/kubeconfig`) or a secret of a service
+    account.  Latter is recommended because it is persistent while the former one is periodically revoked.  Example:
+    type `kubectl create sa auditree_account`) to create a new service account for auditree processes, bind it with
+    appropriate role/clusterrole by rolebinding/clusterrolebinding, and then type
+    `kubectl get secret auditree_account_XXXX -o json | jq .data.token` to get the token where `XXXX` is a random string.
+
+    * `XXX_token`: token string for account `XXX`.
+    * Example credential file entry:
+
+      ```ini
+      [kubernetes]
+      devcluster1_token=your-kubernetes-api-token-1
+      devcluster2_token=your-kubernetes-api-token-2
+      ```
```

Comment:
You also lost me when you went from `XXX` to `devcluster1`.  You should clean this up and make it more clear.

Reply:
for this, I updated the description as follows.
https://github.com/tmishina/auditree-arboretum/commit/df1c2c077982aae57707287f97d77416a610b952